model/src/main/java/org/cbioportal/model/summary/MutationSummary.java

```diff
@@ -0,0 +1,259 @@
+package org.cbioportal.model.summary;
+
+import java.io.Serializable;
+
+public abstract class MutationSummary implements Serializable {
```

Comment:
I read Martin Fowler's `Patterns of Enterprise Application Architecture` book a few weeks ago, and I find Transaction Scripts in Service Layer is a good fit for cBioPortal because the domain logic is not that complicated. Martin Fowler says he doesn't like it but he admits that there are other experts that like it, he says he is an `Object Bigot` but I prefer being pragmatic. There is always a trade-off when you choose a technique over another one, and I try not to be a perfectionist and keep the project simple, easy to understand by everyone in the group. And that's why I'm trying to use the same models for every layer, because I worked in lots of projects that had different models for different layers, those models were very similar but you needed deal with lots of boilerplate code to convert one model to another (I know there are tools like http://mapstruct.org/, but still). Since we have Mixins that we can use to modify the presentation, and since we use MyBatis and manually map the columns to properties(and it's better to have embedded objects if we want to join tables and get extra data), I don't see a problem with using the same models everywhere for now.